portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -98,4 +135,53 @@ private String toJsonString(ArrayList<Comment> commentList){
 
     }
 
+    private String emojify(float score){
+
+        if(score >= -0.1 && score <= 0.1){
```

Comment:
It will make things more obvious, easier to maintain & update in the future, and less susceptible to typos if you set constants for these within your class, i.e.

```
private static final double NEUTRAL_THRESHOLD = 0.1;
private static final double SLIGHTLY_THRESHOLD = 0.4;
private static final double VERY_THRESHOLD = 0.8;
```
Then you can use these constants as follows:
```
if (score >= -NEUTRAL_THRESHOLD && score <= NEUTRAL_THRESHOLD) {
   // Neutral
   return <neutral_emoji>;
}
...
```

Reply:
Changed it! It makes more sense now and its easier to maintain or update!